### DISCUSSION TITLE
Nextcloud

### DISCUSSION THREAD

**Discussion by Maggo1977** [community/community#125998] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello,

I ran the command to install a Nextcloud LXC container on Proxmox.
Where can I find the login details for Nextcloud?

**Comment by queenofcorgis** [comment]
Hi @Maggo1977 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
